main.tex

```diff
@@ -252,8 +251,6 @@ \section*{Abkürzungsverzeichnis}
 \blankpage
 
 \phantomsection
-\pagenumbering{Roman} % Uppercase Roman numerals
-\setcounter{page}{5}
```

Comment:
Das Thesishandbuch sagt wiederum:

"Die Einleitung sollte die erste Seite mit einer arabischen Nummerierung sein, beginnend mit der „1“. Nur die
einzelnen Kapitel im Text der Arbeit werden durchnummeriert, die sonstigen Bestandteile der Arbeit wie
Abbildungsverzeichnis oder Literaturverzeichnis werden ohne Nummerierung angegeben. Der Korpus der Thesis
wird mit arabischen Zahlen nummeriert (1, 2, 3 etc.). Diese Seitenzahlen werden bis zum Schluss weitergeführt."

Reply:
Ich habe folgendes in Introduction to Academic Work gefunden (liegt mir nur in Englisch vor):

Allgemein:

"Apart from the title page, all pages must be numbered. The pages before the body of the
text (e.g., title page, table of contents, list of tables and abbreviations) should be num-
bered in Roman capital letters (I, II, III, IV, etc.), with the page number not appearing on
page I (title page). The pages of the text part are numbered with Arabic numbers (1, 2, 3, etc.), beginning with page 1. This is usually the bullet point “1 Introduction”. These page
numbers are continued to the end, i.e., also through the appendix. The desired position of
the page numbers is centered at the end of the page. Further form specifications can be
found in the Exam Guide in myCampus for every type of examination."


Jedoch für Case Study, Thesis, etc.:

"Only individual chapters in the text are numbered consecutively;
otherwise, sections of the assignment, such as the list of figures or
the bibliography, are not numbered."


Es scheint hier zwei Definitionen zu geben bezgl. der Nummerierung der Seiten nach den einzelnen Kapiteln der Arbeit.
Ich finde jedoch keine Definition basierend auf welcher die Seiten nach dem Text der Arbeit Römisch weitergeführt werden.

Somit denke ich sollte dies entfernt oder durch code ersetzt werden welcher die Nummerierung stopped und bei Bedarf auskommentiert werden kann.